aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -735,4 +802,17 @@ Tensor _sparse_csr_prod_cuda(const Tensor& input, IntArrayRef dims_to_reduce, bo
   return result;
 }
 
+Tensor _sparse_csr_linear_solve(const Tensor& A, const Tensor& b, const bool left) {
+  Tensor b_copy;
+  if (!b.stride(0) == 1) {
+    b_copy = b.clone(c10::MemoryFormat::Contiguous);
+  } else {
+    b_copy = b;
+  }
```

Comment:
Nit: We don't need a conditional here. `Tensor b_copy = b.contiguous()` will be a no op if the tensor is already contiguous.

Reply:
Updated in https://github.com/pytorch/pytorch/pull/129856/commits/882d5a6637edb6fd419a0f5c0ae80f064d6df12d